s2sml/torch_s2s_dataset.py

```diff
@@ -277,13 +275,18 @@ def zscore_values(self, mnv, stdv):
         # if mean and standard deviation are NOT provided do this (only era5)
         if mnv == None or stdv == None:
         
-            self.mean_val = xr.open_mfdataset(
-                self.list_of_era5, concat_dim='sample', combine='nested')[var].mean(
-                skipna=True).values
-
-            self.std_val = xr.open_mfdataset(
-                self.list_of_era5, concat_dim='sample', combine='nested')[var].std(
-                skipna=True).values
+            tmp = xr.open_mfdataset(self.list_of_era5, concat_dim='sample', combine='nested')[var]
+            tmp = self.box_cutter(tmp)
+            
+            if not self.norm_pixel:
+                
+                self.mean_val = tmp.mean(skipna=True).values
```

Comment:
Is this where we should do a weighted mean? See [this page](https://docs.xarray.dev/en/stable/examples/area_weighted_temperature.html) for a nice example using `xarray.DataArray.weighted`

Reply:
Oh right! I forgot about the weighting!

Reply:
good point! we should add this -- so far, this should not be an issue if using small region, but should be changed to be weighted in case we want to train with larger region